cnames_active.js

```diff
@@ -1918,6 +1918,7 @@ var cnames_active = {
   "random": "yuu0007.github.io/random.api",
   "random-machine": "officialpiyush.github.io/random-machine",
   "randy": "randyviandaputra.github.io",
+  "random-stuff-api": "pgamerxstudio.github.io/Random-stuff-api/"
```

Comment:
```suggestion
  "random-stuff-api": "pgamerxstudio.github.io/Random-stuff-api",
  "randy": "randyviandaputra.github.io",
```
Alphabetical order, remove trailing slash, add missing comma.